### PR TITLE
231730: Updated helm charts for ADP and added ADO build pipeline

### PR DESCRIPTION
**Pull request by jeevankuduvaravindran**
This pr includes the changes to update helm charts for ADP and added ADO build pipeline.

https://dev.azure.com/defragovuk/DEFRA-FFC/_workitems/edit/231730

### CODE REVIEW

**Review by kbabigumira — APPROVED** [review]

File: .azuredevops/build.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      ref: refs/heads/main
+
+extends:
+  template: /pipelines/common-app-build.yaml@DEFRA-ADPPipelineCommon
+  parameters:
+    projectName: "FFCDemoPaymentService"
+    deployFromFeature: ${{ parameters.deployFromFeature }}
+    appBuildConfig:
+      appFrameworkType: "nodejs"    
+      projectPath: "./package.json"
```
Comment (DaRoszko) [inline .azuredevops/build.yaml:43]: Can projectPath, Manifest path, and test filePath be defaulted, and no required to be supplied, and enforced/encouraged by convention, to further reduce the size of this file to bare minimum? @jeevankuduvaravindran  @kbabigumira  @rajeshkaremane  @johnwatson484

File: helm/ffc-demo-payment-service/values.yaml [hunk 2]
```diff
@@ -70,4 +70,8 @@ livenessProbe:
   periodSeconds: 10
   failureThreshold: 3
 
-aadPodIdentity: true
+aadPodIdentity: false
```
Comment (DaRoszko) [inline helm/ffc-demo-payment-service/values.yaml:73]: Can we just kill off podIdentity reference?

File: .azuredevops/build.yaml (hunk 1 above)
Comment (rajeshkaremane) [reply .azuredevops/build.yaml:43]: Will take it as separate story, if that is fine

File: helm/ffc-demo-payment-service/values.yaml (hunk 2 above)
Comment (jeevankuduvaravindran) [reply helm/ffc-demo-payment-service/values.yaml]: @DaRoszko Done

**Review by kbabigumira — APPROVED** [review]

**Pull request merged**